packages/terra-heading/docs/heading-sizes.md

```diff
@@ -0,0 +1,3 @@
+## Heading Sizes
+
+The `size` prop can be used to alter the default size of any `<Heading />` component whenever necessary.
```

Comment:
I know these options are list in the props, but I think it would be valuable to list them again here (& in the heading weights section) so it's clear on the site page.